### ISSUE TITLE
A question on simple_switch_grpc and simple_switch_CLI

### ISSUE THREAD

**Issue by LwbXc** [p4lang/behavioral-model#874]
I create a switch by using simple_switch_grpc, but there is an error when I try to use simple_switch_CLI to connect to it.

Here is the code how I create a switch:
sdn@onos-p4-tutorial:~/Sonata$ simple_switch_grpc --thrift-port 22222 ~/bmv2/mininet/simple_router.json 
Calling target program-options parser
I0314 23:10:08.927065906   27989 server_builder.cc:247]      Synchronous server. Num CQs: 1, Min pollers: 1, Max Pollers: 2147483647, CQ timeout (msec): 1000
Server listening on 0.0.0.0:50051
Thrift server was started

And here is the code how I try to use simple_switch_CLI to connect to it:
sdn@onos-p4-tutorial:~/Sonata$ simple_switch_CLI --thrift-port 22222
Obtaining JSON from switch...
Error when requesting JSON config from switch

Maybe I misunderstand the usage of them, how should I use a CLI to control a simple_switch_grpc switch?

**Comment by jafingerhut** [comment]
I tried similar commands, with a different BMv2 JSON compiled program than
you used, on my system, and did not see such an error.  Here are the
versions of p4c and simple_switch_grpc I have installed on my system.  You
can run those commands on yours to see what versions you have, in case that
might be part of the cause.

How did you install these tools on the system you are using?

$ p4c --version
p4c 1.2.0 (SHA: 62fd40a0)

$ simple_switch_grpc --version
1.13.0-c88dc2de

Andy
